test/compilable/staticforeach.d

```diff
@@ -120,6 +120,7 @@ static foreach(member;__traits(allMembers,S)){
 }
 
 // print prime numbers using overload sets as state variables.
+/+
 static assert(is(typeof(bad57)));
 static assert(!is(typeof(bad53)));
```

Comment:
I'm not sure how I can "fix" this test. It generated `bad57` a couple of times and AFAICT there's no `break` for `static foreach` yet.

CC @tgehr